app/forms/add_to_order_form.rb

```diff
@@ -219,9 +220,8 @@ def order_status
   def merge_order
```

Comment:
I think we can do our future selves a favor by adding some code comments here about what this method actually does/returns.  I would also support renaming to `target_order` or something else that is more revealing of intention.